cmd/start.go

```diff
@@ -46,5 +46,6 @@ func init() {
 	cmd.PersistentFlags().BoolVar(&opts.PromQLTarget.UseDNS, "use-dns-for-services", opts.PromQLTarget.UseDNS, "Configures the CVO to use DNS for resolution of services in the cluster.")
 	cmd.PersistentFlags().StringVar(&opts.PrometheusURLString, "metrics-url", opts.PrometheusURLString, "The URL used to access the remote PromQL query service.")
 	cmd.PersistentFlags().BoolVar(&opts.InjectClusterIdIntoPromQL, "hypershift", opts.InjectClusterIdIntoPromQL, "This options indicates whether the CVO is running inside a hosted control plane.")
+	cmd.PersistentFlags().StringVar(&opts.Upstream, "upstream", opts.Upstream, "The preferred update server.  If set, this option overrides any values configured in ClusterVersion spec.")
```

Comment:
I'm not sure whether `--upstream` makes enough sense for someone eyeballing the future CVO manifest and wondering what it does (upstream makes at least a little sense in the context of the CV resource. 

Thoughts on explicit `--update-service`? We'd lost consistency between CV and CVO manifest but that assume folks are actually familiar with CV resource.

Reply:
I've pushed 64fb2264 -> 04c32b52 with a pivot to `--update-service`, `updateService`, etc.